src/Nowruz/modules/settings/components/orgTeam/useOrgTeam.tsx

```diff
@@ -0,0 +1,161 @@
+import { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  CurrentIdentity,
+  Following,
+  addOrganizationMember,
+  filterFollowings,
+  getOrganizationMembers,
+  removeOrganizationMember,
+} from 'src/core/api';
+import { Avatar } from 'src/Nowruz/modules/general/components/avatar/avatar';
+import { AccountItem } from 'src/Nowruz/modules/general/components/avatarDropDown/avatarDropDown.types';
+import { RootState } from 'src/store';
+
+export const useOrgTeam = () => {
+  const currentIdentity = useSelector<RootState, CurrentIdentity | undefined>((state) =>
+    state.identity.entities.find((identity) => identity.current),
+  );
+
+  const [addedMembers, setAddedMembers] = useState<string[]>([]);
+  const [teamMembers, setTeamMembers] = useState<AccountItem[]>([]);
+  const [selectedPerson, setselectedPerson] = useState();
+  const [openAlert, setOpenAlert] = useState(false);
+  const [toDeleteName, setToDeleteName] = useState('');
+  const [toDeleteId, settoDeleteId] = useState('');
+  const [page, setPage] = useState(1);
+  const [hasMore, setHasMore] = useState(true);
+
+  const getTeamMembers = async () => {
+    try {
+      const res = await getOrganizationMembers(currentIdentity!.id, { page: 1 });
+      const mapped = res.items.map((item) => {
```

Comment:
what about if our `res` doesn't have `.items` or empty `.items` from BE ?

Reply:
line-53 too

Reply:
Logically of org can login to the system, it definitely has at least one member. However, I added ? to avoid error,

Reply:
Thanks